### PR TITLE
Add DEV cassandra IP to conf file.

### PR DESCRIPTION
**Pull request by novakov-alexey**

### CODE REVIEW

**Comment by novakov-alexey** [comment]
merging, but having constant hostname "cassandra"

File: server/app/dal/CassandraCluster.scala [hunk 1]
```diff
@@ -44,14 +42,14 @@ class CassandraConfig @Inject()(configuration: Configuration, environment: Envir
     val hostsConf = "devgym.db.cassandra.hosts"
     val dockerConf = "devgym.db.cassandra.docker"
 
-    def hosts: Seq[String] = {
+    def getHosts: Seq[String] = {
```
Comment (DmytroOrlov) [inline server/app/dal/CassandraCluster.scala]: java style?

File: server/app/dal/CassandraCluster.scala [hunk 2]
```diff
@@ -23,13 +23,11 @@ class CassandraCluster @Inject()(conf: CassandraConfig, appLifecycle: Applicatio
...
 
   Logger.info(s"Cassandra host to be used: ${hosts.mkString(",")}:${conf.port}")
+  appLifecycle.addStopHook(() => stop())
 
   def noKeySpaceSession: Session = cluster.connect()
   def session: Session = cluster.connect(conf.keySpace)
-
   def stop() = toFutureUnit(cluster.closeAsync())
-
-  appLifecycle.addStopHook(() => stop())
```
Comment (DmytroOrlov) [inline server/app/dal/CassandraCluster.scala]: code at the bottom of method is more convenient for scala

File: server/app/dal/CassandraCluster.scala [hunk 3]
```diff
@@ -44,14 +42,14 @@ class CassandraConfig @Inject()(configuration: Configuration, environment: Envir
     val hostsConf = "devgym.db.cassandra.hosts"
     val dockerConf = "devgym.db.cassandra.docker"
```
Comment (DmytroOrlov) [inline server/app/dal/CassandraCluster.scala]: this is junk that I forgot to delete

File: server/app/dal/CassandraCluster.scala (hunk 1 above)
Comment (novakov-alexey) [inline server/app/dal/CassandraCluster.scala]: yeah, there were words "hosts", kind of kalambur and hard to read. If you want you can change name back.

File: server/app/dal/CassandraCluster.scala (hunk 2 above)
Comment (novakov-alexey) [inline server/app/dal/CassandraCluster.scala]: Ok. Then let's move Logger.info and that string to bottom.

**Pull request merged**